### DISCUSSION TITLE
Clarification about benchmark dataset

### DISCUSSION THREAD

**Discussion by viniciusmgn** [NVlabs/curobo#179] [Research Q&A] [answered]
Hello. My question is related to the robometrics dataset in "https://github.com/fishbotics/robometrics". They don't have a discussion there, so I will ask here since it is also closely related to cuRobo.

I would like to understand better the structure of the dataset. 

(1) - What is the meaning of the "collision_buffer_ik" field?
(2) - What is the meaning of the "goal_ik" field?
(3) - The field "start" has, I suppose, Franka Emika Panda's starting configuration. But according to which convention? Can you please provide the DH table that was used? Furthermore, if the DH convention is Craig's (modified) or Spong's (standard)?

**Comment by balakumar-s** [comment, marked as answer]
1. ``collision_buffer_ik`` is the penetration depth between the robot and the world at the goal pose. We added this field as for a very small number of problems, the goal pose was in collision with the world. This is because of our approximation of the robot with spheres. If this distance is < 0, then we don't benchmark that problem as seen here: https://github.com/NVlabs/curobo/blob/b481ee201a2754c072ba9bcd80bd5d327fddc599/benchmark/curobo_benchmark.py#L301
2. ``goal_ik`` gives valid joint configurations at the goal pose. We provide this as some planners don't take as input a target Cartesian pose. 
3. We did not use DH parameters for the Franka Panda. We read it from this urdf: https://github.com/NVlabs/curobo/blob/main/src/curobo/content/assets/robot/franka_description/franka_panda.urdf with the kinematic chain from ``base_link`` to ``panda_hand``. There might be some tools online to convert URDF to dh like this one: https://github.com/mcevoyandy/urdf_to_dh/tree/main
